raft-tech/TANF-app
# [WIP] 2154 assign filename ssp section after uploading file

## Summary of Changes
After uploading files, the filenames are changed based on assigned section using provided associated filename. Currently, the logic for renaming files for SSP section is missing and uploaded files are renamed incorrectly for SSP files.

Pull request closes #2154

## How to Test

The following steps skip how to start and run the server (either local or deployed in cloud.gov)

1. Login to TDRS
2. Make sure your user has the correct rights
3. (This step requires SSP checkbox on the frontend) Upload a file with SSP checked
4. Check filename is correct per [States filename table](https://github.com/raft-tech/TANF-app/blob/develop/tdrs-backend/tdpservice/stts/management/commands/data/states.csv)

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
